src/Cassandra/Data/Linq/Batch.cs

```diff
@@ -68,15 +68,29 @@ public Batch Append(IEnumerable<CqlCommand> cqlCommands)
 
         public void Execute()
         {
-            EndExecute(BeginExecute(null, null));
+            var queryAbortTimeout = _session.Cluster.Configuration.DefaultRequestOptions.QueryAbortTimeout;
```

Comment:
We could use this opportunity to make an internal reusable property whenever possible.

Reply:
I added a property here but it's not reusable anywhere else currently.